app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -357,6 +371,61 @@ private fun MessageStatusLabel(messageStatus: MessageStatus) {
     }
 }
 
+@Composable
+private fun MessageSentPartialDeliveryFailures(partialDeliveryFailureContent: DeliveryStatusContent.PartialDelivery) {
+    val resources = LocalContext.current.resources
+    var expanded: Boolean by remember { mutableStateOf(false) }
+    CompositionLocalProvider(
+        LocalTextStyle provides MaterialTheme.typography.labelSmall.copy(color = MaterialTheme.wireColorScheme.error)
+    ) {
+        Column {
+            Text(
+                text = stringResource(
+                    id = R.string.label_message_partial_delivery_participants_count,
+                    partialDeliveryFailureContent.totalUsersWithFailures
+                ),
+                textAlign = TextAlign.Start
+            )
+            VerticalSpace.x4()
+            if (expanded) {
+                if (partialDeliveryFailureContent.noClients.isNotEmpty()) {
+                    Text(
+                        text = stringResource(
+                            id = R.string.label_message_partial_delivery_participants_wont_deliver,
+                            partialDeliveryFailureContent.noClients.joinToString(", ") { it.asString(resources) }
+                        ),
+                        textAlign = TextAlign.Start
+                    )
+                }
+                if (partialDeliveryFailureContent.failedRecipients.isNotEmpty()) {
+                    Text(
+                        text = stringResource(
+                            id = R.string.label_message_partial_delivery_participants_deliver_later,
+                            partialDeliveryFailureContent.failedRecipients.joinToString(", ") { it.asString(resources) }
+                        ),
+                        textAlign = TextAlign.Start
+                    )
+                }
```

Comment:
I had a look at this, and prefer to leave it as it is right now, because these are 2 text built from different column sources in db. Also, there is a current discussion in federation team about these texts 🙈

I'll add a `TODO` for adding the animation later and hide this under a feature flag, even tough this will not happen until the backend deploys v3